ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -379,6 +379,30 @@ ${local.skip_on_weekends}
   }
 }
 
+resource "azurerm_monitor_metric_alert" "function_app_memory_alert" {
+  count               = var.function_id == null || contains(var.disabled_alerts, "function_app_memory_alert") ? 0 : 1
```

Comment:
Just for my own learning what that the count field do? (as it seems to be the one that is turning on/off the alert) and what is the difference with enabled field? 🤔